hazelcast/src/main/java/com/hazelcast/nio/tcp/AbstractIOSelector.java

```diff
@@ -95,6 +99,11 @@ public final void addTask(Runnable runnable) {
         selectorQueue.add(runnable);
     }
 
+    @Probe
+    private long idleTime() {
+        return Math.abs(System.currentTimeMillis() - lastSelectTimeMs);
```

Comment:
Peter, how is 0.56% improvement possible while you are introducing additional load for counting etc?